README.md

```diff
@@ -38,45 +38,59 @@ To run:
 
 ### analysis.py
 
-Input : CYCLUS output file (.sqlite)
-`python analysis.py [outputfile]`
+Input : CYCLUS output file (.sqlite)  
+```
+python analysis.py [outputfile]
+```
 
 Most functions return a dictionary of lists (timeseries of a value)
 that can be used to plot a stacked bar chart or a line plot.
 
 Major analysis functions include:
-1. commodity_in_out_facility
+1. facility_commodity_flux  
 Returns dictionary of influx/outflux of a commodity from a facility.
 
-2.  get_stockpile
+2.  get_stockpile  
```

Comment:
while it is very nice to have this stuff in the readme, I just want to note that these function definitions aren't necessary here. If they are in docstrings in the analysis.py file already, you can access them with the question mark in the python interpreter (or in a notebook, or with sphinx).